resources/examples/manager/BasicAssetLibrary/tests/bal_business_logic_suite.py

```diff
@@ -53,6 +55,53 @@ def test_returns_ignored_policy_for_write_for_all_trait_sets(self):
             self.assertEqual(policy, constants.kIgnored)
 
 
+class Test_managementPolicy_library_specified_behavior(FixtureAugmentedTestCase):
+    """
+    Tests that custom policies are loaded and respected.
+    """
+
+    __trait_sets = (
+        {"definitely", "unique"},
+        {"an", "ignored", "trait", "set"},
+        {"a", "non", "exclusive", "trait", "set"},
+    )
+
+    __expected_policies = [4, 0, 1]
+
+    # We need to load a different library for these tests. This is
+    # a little fiddly as the manager isn't (currently) recreated
+    # for each test.
+
+    ## @todo Extend openassetio.test.manager to allow settings to be
+    ## varied per test class/method.
+
+    def setUp(self):
+        self.__old_settings = self._manager.getSettings()
+        new_settings = self.__old_settings.copy()
+        new_settings["library_path"] = os.path.join(
+            os.path.dirname(__file__),
+            "resources",
+            "library_business_logic_suite_customManagementPolicy.json",
+        )
+        self._manager.setSettings(new_settings)
+        self._manager.initialize()
```

Comment:
> Here, I take it we assume setSettings is lightweight and we must re-initialize the manager to act on them. But we happen to know the internals of the manager, which isn't the general case.

I vaguely remember that the intention was definitely that, settings are only applied when `initialize` is called. So if you change settings, you have to re-intialize. If we haven't documented that we def need to, ill look into it.

> whether a ManagerInterface.tearDown/shutDown/cleanUp/terminate/destroy method should be added

Do we think it adds any specific merit over the destructor? The reason for `initialize` is that we need to be able to cheaply create an instance for introspection, before making it ready for use. I wonder if adding a `reinitialize` method or similar complicates lifecycle management further?

Reply:
> setSettings behaviour could be conditional, only re-loading the database if already initialized (which would allow us to get rid of initialize() here)?

The only concern there was that it means that there is more logic in the manager plugin. I think before we quite liked that the settings mechanism is a simple update of the value, and the work happens in `initialize`.

Reply:
> I vaguely remember that the intention was definitely that, settings are only applied when `initialize` is called. So if you change settings, you have to re-intialize. If we haven't documented that we def need to, ill look into it.

```
    def setSettings(self, settings, hostSession):
        """
        @todo Document settings mechanism
        """
```

> Do we think it adds any specific merit over the destructor?

Fine for C++, and maybe Python could use `__del__` (though ugly) - C API is trickier.

Perhaps a `reinitialize` method?